washing-dishes.md

```diff
@@ -0,0 +1 @@
+# washing dishes
```

Comment:
Every thing is looking good,but you should had put "my" before the word PR.You should have use a dash rather than a space.